### ISSUE TITLE
[feat] Agenda

### ISSUE THREAD

**Issue by marionnegp** [RedHat-UX/red-hat-design-system#329]
### Description

The agenda component/pattern would be an updated version of the [agenda in FTS](https://webrh-demo.apps.int.spoke.preprod.us-west-2.aws.paas.redhat.com/schema/agenda).

### Acceptance Criteria

- [ ] Design done
- [ ] Development done

### Other resources

**Notes from Trello:**
- XD link: https://xd.adobe.com/view/d2e0c019-6aa5-4bc6-aaed-3aa59e0cee69-87b3/specs/
- Collapsed view vs. expanded? https://www.redhat.com/en/events/webinar/develop-deploy-deliver-continuously#agenda vs. https://www.redhat.com/en/summit/agenda/agenda-at-a-glance (Dan)
- For the first link, wondering if a series of Accordions can be used instead. For the second link, maybe the Alert pattern can be tweaked or a new Card style with a left border can be created. (@coreyvickery)
- what's missing from that example is the time beside the "accordion" piece: https://webrh-patternkit.int.open.paas.redhat.com/schema/daily_agenda (Dan)

**Additional examples from event design team:**

- [Summit Boston 2022 ](https://www.redhat.com/en/summit/boston)
- [Summit Connect: Raleigh 2021](https://www.redhat.com/en/summit/connect/raleigh)
- [Summit Connect: Germany 2021](https://www-admin.corp.redhat.com/de/summit/connect/emea/germany)
- [Test example 1](https://www-admin.corp.redhat.com/en/summit/connect/emea/template#)
- [Test example 2](https://www-admin.corp.redhat.com/en/forums-2021-template-examples-wip)

**Comment by marionnegp** [comment]
Feedback from Adam Thornburg for implementation in Drupal:

This is more a note for the development but the current abstract and speaker info section in drupal are not flexible. You can't add bulleted list, line breaks, links, etc. I would want this section to be WYSIWYG editor.

**Comment by marionnegp** [comment]
Latest version after making edits from Adam and other event designers:

![Image](https://user-images.githubusercontent.com/95588923/185479909-829b0aee-8a4f-4d2f-8b8d-35de06327722.png)

**Comment by coreyvickery** [comment]
@marionnegp Did you explore mobile examples?

**Comment by marionnegp** [comment]
Yep, I did one for a [simple agenda on mobile](https://xd.adobe.com/view/2366a943-b81b-4af4-b74f-2a7cb5f496bd-7635/screen/9bba57be-71c3-4330-8d73-315a8639339e) and [one with nested events](https://xd.adobe.com/view/2366a943-b81b-4af4-b74f-2a7cb5f496bd-7635/screen/93710ac0-2218-4b9b-af40-aeefeca0da40/).

**Comment by marionnegp** [comment]
Designers for events prefer to have the location and description always visible for T1 events' sessions. I'm working on options for showing event descriptions.

Here are [options for keeping the location visible](https://xd.adobe.com/view/784e4a4e-4bf5-4d6b-b5fe-c4f7ce7cc347-2d1d/).

**Comment by marionnegp** [comment]
Updated agenda link: https://xd.adobe.com/view/b34fba7f-a7aa-4ccc-acf3-7e1ff7e9547f-e61c/
- Collapsed agenda accordion shows location now
- I've added a version of the agenda for T1 events only. Event designers said that they prefer an agenda that shows all the info without accordions for T1 events because the agenda doesn't usually show a lot of info, and content will not differ much from event to event.


![Image](https://user-images.githubusercontent.com/95588923/220387543-a5070cf0-5c80-4cc2-913d-3c6f7f9c65b7.png)

**Comment by marionnegp** [comment]
I added an example of a session with a long title and tags to page 2 of the [latest XD link](https://xd.adobe.com/view/b34fba7f-a7aa-4ccc-acf3-7e1ff7e9547f-e61c/) and mobile examples for the regular event agenda and the T1 agenda. Event designers said this looks good to them.

**Comment by markcaron** [comment]
Keeping open for now. Will need to revisit once it's a priority.

CC @marionnegp

**Comment by bennypowers** [comment]
seems more like a pattern at first glance...

**Comment by coreyvickery** [comment]
@markcaron @bennypowers I don't think this should be part of Cubone, maybe Diglett.

**Comment by markcaron** [comment]
@marionnegp do we need this as a component anymore?

**Comment by coreyvickery** [comment]
@markcaron No idea on my end, but I'll let @marionnegp answer too.

**Comment by marionnegp** [comment]
I haven't heard about a need for this in a long time. I'd just close it unless we get a new request.
